packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.24
+
+* Add support for building `WebView` widget with Android hybrid views. To use this feature, set
+  `WebView.platform` to an instance of `SurfaceAndroidWebView`.
```

Comment:
What about this (note the link to hybrid composition and example):


* Add support for building `WebView` widget with [Android hybrid views](https://github.com/flutter/flutter/wiki/Hybrid-Composition).
  To use this feature, set `WebView.platform` to an instance of `SurfaceAndroidWebView`. For example:
  
  ```dart
  class WebViewExample extends StatefulWidget {
    @override
    void initState() {
      super.initState();
      WebView.platform = SurfaceAndroidWebView();
    }

    @override
    Widget build(BuildContext context) {
      return WebView(
        initialUrl: 'https://flutter.dev',
      );
    }
  }
  ```